mysql-test/suite/rocksdb/r/index_merge_rocksdb2.result

```diff
@@ -535,11 +536,10 @@ key(cola), key(colb)
 INSERT INTO t1 VALUES ('foo','bar', 'ZZ'),('fuz','baz', 'ZZ');
 OPTIMIZE TABLE t1;
 Table	Op	Msg_type	Msg_text
-test.t1	optimize	note	Table does not support optimize, doing recreate + analyze instead
 test.t1	optimize	status	OK
 select count(*) from t1;
 count(*)
-8704
+526
```

Comment:
Say what?

Reply:
Actually, neither 526 nor 8704 make any sense:

> # BUG#16166
> CREATE TABLE t1 (
> cola char(3) not null, colb char(3) not null,  filler char(200),
> key(cola), key(colb)
> );
> INSERT INTO t1 VALUES ('foo','bar', 'ZZ'),('fuz','baz', 'ZZ');
> OPTIMIZE TABLE t1;
> Table	Op	Msg_type	Msg_text
> test.t1	optimize	status	OK
> select count(*) from t1;
> count(*)
> 526
To me it looks like the result should be "2" all day long from any engine, but, both InnoDB and MyISAM return "8704" and MyRocks is returning "526" :-\

Reply:
Nevermind, this is just from the result, the test itself disables the query log and inserts a bunch of rows.  So yeah, MyRocks is doing something wrong here, but for some reason only in 8.0 (or it's possible this is a new test bit in 8.0).

Reply:
This is actually another utf8mb4 secondary key bug https://jira.percona.com/browse/PS-5024